src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -15,29 +15,18 @@
     public static class BuiltInCultureConventions

```

Comment:
Why don't we make this an instance and pass in `GlobalizationConfiguration` instead of passing it to every method?